code/_helpers/text.dm

```diff
@@ -1,3 +1,5 @@
+#define SMALL_FONTS(FONTSIZE, MSG) "<span style=\"font-family: 'Small Fonts'; -dm-text-outline: 1 black; font-size: [FONTSIZE];\">[MSG]</span>"
```

Comment:
Is the font family `Small Fonts` actually defined anywhere or is that some weirdness that's been copied over and over?

Reply:
it's an in-built font , i think. i don't know enough to mess with it, but it looks damn good, so i use it everywhere

Reply:
Interesting, can't find any documentation on it. Are you _sure_ it's not just the default font rendering in there? 😰